packages/service-library/src/servicelib/aiohttp/long_running_tasks/_error_handlers.py

```diff
@@ -7,18 +9,22 @@
     TaskNotFoundError,
 )
 
+_logger = logging.getLogger(__name__)
+
 
 @web.middleware
 async def base_long_running_error_handler(request, handler):
     try:
         return await handler(request)
     except (TaskNotFoundError, TaskNotCompletedError) as exc:
+        _logger.debug("", exc_info=True)
```

Comment:
No, I was testing a bit and it when logging here (meaning when handling the exception rather than raising it) it is much more meaningful to log the `exc_info` rather than the stack. I think here the stack will not show anything related to the calls which produced the exception, but only calls higher/lower in the stack.